src/database/MySQLTable.ts

```diff
@@ -25,7 +26,15 @@ export class MySQLTable {
   public async apply(row: ParsedRecord): Promise<void> {
     this.buffer[row.action].push(row);
 
-    if (this.buffer[row.action].length >= this.flushLimit) {
+    if (row.action === RecordAction.DelayedInsert) {
+      const values = Object
+        .entries(row.values)
```

Comment:
I think you should only use the key members, not all the values, because if they update any BS values it's not going to find it.